test/integration-tests/strategies/deltaStrategy.ts

```diff
@@ -306,31 +306,84 @@ describe('Delta Strategy integration test', async () => {
     });
   });
   describe('start round 2', async () => {
+    let strikes: BigNumber[] = [];
+    before('prepare before new round start', async () => {
+      // set price back to initial spot price
+      await TestSystem.marketActions.mockPrice(lyraTestSystem, spotPrice, 'sETH');
+
+      // initiate withdraw for later test
+      await vault.connect(randomUser2).initiateWithdraw(toBN('50'));
+    });
     before('create new board', async () => {
       await TestSystem.marketActions.createBoard(lyraTestSystem, boardParameter);
       const boards = await lyraTestSystem.optionMarket.getLiveBoards();
       boardId = boards[0];
+
+      strikes = await lyraTestSystem.optionMarket.getBoardStrikes(boardId);
     });
     it('start the next round', async () => {
       await lyraEvm.fastForward(lyraConstants.DAY_SEC);
       await vault.connect(manager).startNextRound(boardId);
     });
-    // it('should be able to complete the withdraw', async() => {
-    //   const sethBefore = await seth.balanceOf(randomUser2.address)
+    it('should be able to complete the withdraw', async () => {
+      const sethBefore = await seth.balanceOf(randomUser2.address);
+
+      await vault.connect(randomUser2).completeWithdraw();
 
-    //   await vault.connect(randomUser2).completeWithdraw();
+      const sethAfter = await seth.balanceOf(randomUser2.address);
 
-    //   const sethAfter = await seth.balanceOf(randomUser2.address)
+      expect(sethAfter.sub(sethBefore).gt(toBN('50'))).to.be.true;
+    });
+    it('should be able to trade', async () => {
+      const strategySUSDBalanceBefore = await susd.balanceOf(strategy.address);
+      // 3400 is a good strike
+      await vault.connect(randomUser).trade(strikes[3]);
+      const strategySUDCBalanceAfter = await susd.balanceOf(strategy.address);
+      expect(strategySUDCBalanceAfter.sub(strategySUSDBalanceBefore).gt(0)).to.be.true;
+    });
+    it('should revert when trying to reduce a safe position', async () => {
+      const positionId = await strategy.strikeToPositionId(strikes[3]);
+      await expect(vault.connect(randomUser).reducePosition(positionId)).to.be.revertedWith(
+        'position properly collateralized',
+      );
 
-    //   console.log(sethAfter.sub(sethBefore).toString())
-    // })
+      // we remain safe even if eth goes to 3900 (30%)
+      await TestSystem.marketActions.mockPrice(lyraTestSystem, toBN('3900'), 'sETH');
+      await expect(vault.connect(randomUser).reducePosition(positionId)).to.be.revertedWith(
+        'position properly collateralized',
+      );
+    });
+    it('@todo: fix: cannot use reduce position because delta will be out of range', async () => {
+      await TestSystem.marketActions.mockPrice(lyraTestSystem, toBN('4000'), 'sETH');
+      const positionId = await strategy.strikeToPositionId(strikes[3]); // 3400 strike
+      await expect(vault.connect(randomUser).reducePosition(positionId)).to.be.revertedWith('delta out of range');
+    });
```

Comment:
@joshua0702k here it's the issue that I'm not sure what's the best way to resolve

Reply:
Since our targetCollatPercent is 40%, strategy sets collateral that is way above bufferCollateral. So we require a really big jump in price to make position.collateral < bufferCollateral. With such a big price jump, the strike ends up exceeding delta (on the high side)

I'll push a separate PR with all the fixes for this issue (there are several). Please merge that into this branch.